apps/graphql/src/definitions/Build.js

```diff
@@ -200,6 +200,19 @@ export const resolvers = {
         throw new APIError("Invalid user authorization");
       }
 
+      const build = await Build.query()
+        .findById(buildId)
+        .withGraphFetched("repository");
+
+      if (build.repository.private) {
+        const account = Account.getAccount(build.repository);
+        const screenshotsConsumptionRatio =
+          await account.getScreenshotsConsumptionRatio();
+        if (screenshotsConsumptionRatio >= 1.1) {
```

Comment:
This limit is used several times, put it in a constant please.

Reply:
I'm ok to use a constant but I don't know where to store it. Until I find a good place, I'll stick to hard coded values